src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -192,6 +199,7 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
         super(props);
 
         makeObservable(this);
+        this.mutationPlotRef = React.createRef();
```

Comment:
We probably don't need this line

Reply:
Yes, I was just trying out different things, removed this line